care/facility/migrations/0095_recompute_facility_types.py

```diff
@@ -0,0 +1,64 @@
+# Generated by Django 2.2.11 on 2020-04-15 07:53
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Similar lines in 2 files](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5374455)